crates/swc_ecma_ast/src/expr.rs

```diff
@@ -63,6 +64,10 @@ pub enum Expr {
     #[tag("MemberExpression")]
     Member(MemberExpr),
 
+    #[tag("SuperPropExpression")]
+    #[is(name = "super_")]
+    Super(SuperPropExpr),
```

Comment:
I think `SuperProp` would be better, as `Expr::Super` looks like `super` instead of `super.foo`.

Reply:
If name could remain the same I could push an commit to rename this, and rebase it before update parser change

Reply:
Well the change is not that big, I could handle it mannually

Reply:
I disabled auto-merge of parser PR. Thanks!